modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/RegisteredDomainProcessorTests.java

```diff
@@ -41,6 +41,15 @@ public void testBasic() throws Exception {
             "1"
         );
         testRegisteredDomainProcessor(buildEvent("www.books.amazon.co.uk"), "www.books.amazon.co.uk", "amazon.co.uk", "co.uk", "www.books");
+        testRegisteredDomainProcessor(buildEvent("example.com"), "example.com", "example.com", "com", null);
+        testRegisteredDomainProcessor(buildEvent("googleapis.com"), "googleapis.com", null, "googleapis.com", null);
```

Comment:
Could you move the above `fastly.net` test here and then add a comment to the effect that it, the `googleapis.com` test, and the `content-autofill...` test are testing third-party eTLDs? I think that will clarify the purpose of those tests and illustrate why they behave differently than the other tests.

Reply:
Resolved.